packages/@tinacms/graphql/src/builder/index.ts

```diff
@@ -438,7 +438,16 @@ export class Builder {
       [NAMER.createName([collection.name])]: 'create',
       [NAMER.updateName([collection.name])]: 'update',
     })
-    return astBuilder.FieldDefinition({ type, name, args, required: true })
+    // Is errorPolicy set to include?
+    const errorPolicyInclude =
+      this.config.tinaSchema.config?.config?.client?.errorPolicy === 'include'
+    return astBuilder.FieldDefinition({
+      type,
+      name,
+      args,
+      // If errorPolicy is set to include, we don't want to throw an error so we set required to false
+      required: !errorPolicyInclude,
```

Comment:
Is this change is necessary? If we don't throw from the client it will get stored the error in the `errors` array, which seems like the desired behavior. This would make it so that it's just `null` with no error message anywhere.

Reply:
cc @logan-anderson 

Also, I don't know about having the errorPolicy be apart of the client. Seems like something you may want at the request-level depending on what your fetching.

Reply:
I think it makes sense to be available in both spaces. Having said that, I'd almost wonder if we're better off just never throwing and just return the errors array.

> we can update that do be required: false

That seems fine, though I think leaving it as-is is also ok. If you have an `errors` array then it's safe to say that all bets are off for your Typescript types, IMO. Mainly, I just think we should alter between the 2 behaviors based on the client config.

Reply:
> That seems fine, though I think leaving it as-is is also ok.
> I just think we should not alter between the 2 behaviors based on the client config (though I get why you did that)

So are you saying that we should implement what @logan-anderson is suggesting here? https://github.com/tinacms/tinacms/pull/4223#discussion_r1347453912

Just want to be clear on next steps to get this through

Reply:
Sorry @jamespohalloran yes I was saying that seems good to do. This all looks good to me 👍